db/migrate_cards/20151204144734_import_modified_company_topic_pages.rb

```diff
@@ -0,0 +1,8 @@
+# -*- encoding : utf-8 -*-
+
+class ImportModifiedCompanyTopicPages < Card::Migration
+  def up
+    import_json "modified_company_topic_pages.json"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.